arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -119,19 +122,19 @@ cargo stylus new <YOUR_PROJECT_NAME>
 
 `cargo stylus new` generates a starter template that implements a Rust version of the [Solidity `Counter` smart contract example](https://github.com/OffchainLabs/counter_contract/blob/master/contracts/Counter.sol).
 
-At this point, you can move on to the next step of this guide or develop your first Rust smart contract. Feel free to use the [Stylus Rust SDK reference](./reference/overview) as a starting point, it offers many examples to quickly familiarize yourself with Stylus.
+At this point, you can move on to the next step of this guide or develop your first Rust smart contract. Feel free to use the [Stylus Rust SDK reference section](./reference/overview) as a starting point; it offers many examples to help you quickly familiarize yourself with Stylus.
 
 ## Checking if your Stylus project is valid
 
 By running `cargo stylus check` against your first contract, you can check if your program can be successfully **deployed and activated** on-chain.
 
-**Important:** Ensure your Docker service is running for this command to work correctly.
+**Important:** Ensure your Docker service runs so this command works correctly.
 
 ```shell
 cargo stylus check
 ```
 
-`cargo stylus check` executes a dry-run on your project by compiling your contract to WASM and verifying if it can be deployed and activated onchain.
+`cargo stylus check` executes a dry run on your project by compiling your contract to WASM and verifying if it can be deployed and activated on-chain.
```

Comment:
It's become convention in the industry to gravitate towards `onchain` vs `on-chain` in the same way that we moved from `e-mail` to `email` and `on-line` to `online`.

Reply:
Agreed! But I still get some pushback for it. Fixed.